### ISSUE TITLE
Highlights all tags of the same type rather than just closing/opening tag

### ISSUE THREAD

**Issue by alexchantastic** [mblode/vscode-twig-language-2#1]
When placing the cursor in a tag, all tags of the same type are highlighted rather than just the closing/opening tag.

![image](https://user-images.githubusercontent.com/604167/35469280-9b0bc0c0-02e6-11e8-84fa-3df27133f1a9.png)

This is what I would expect to happen:

![image](https://user-images.githubusercontent.com/604167/35469282-a5a6fc8e-02e6-11e8-8f1e-32808c599021.png)

**Comment by mblode** [comment]
@alexchantastic I see your issue but I believe the problem of highlighting may be accentuated by the VSCode theme you are using. I am personally using One Dark Italic where the all of the other tags are still highlighted but more subtly. I may be able to create a version of your theme where the tag highlighting is hidden if you would like. 

![screen shot 2018-02-14 at 9 17 29 am](https://user-images.githubusercontent.com/7183998/36176902-367f1414-1168-11e8-8eb7-6494953e9702.png)

**Comment by mblode** [comment]
Hey @alexchantastic, I have just pushed an update to Twig Language 2 with a brand new formatter using Prettydiff 3 which brings about some major bug fixes. I tested your issue with a fresh install of VS Code and it seems to be the system default behaviour of highlighting all instances of selected text, a behaviour that also occurs when using Twig Pack extension. Sorry about the slow development speeds and sub-par communication but hopefully these updates have fixed some of your other problems.
